build.gradle

```diff
@@ -31,17 +37,67 @@ buildscript {
 plugins {
     id('io.github.gradle-nexus.publish-plugin') version '1.1.0'
     id('org.sonatype.gradle.plugins.scan') version '2.4.0'
-    id("org.jetbrains.dokka") version "1.7.20"
 }
 
+apply plugin: "org.jetbrains.dokka"
```

Comment:
move this to top of the file

Reply:
I got an error in the build if it appeared before the `buildscript` or `plugins` blocks, so I put it just after those, near the top of the file.